### PR TITLE
LinkedElementsInView, CurveLoop Tessellate, View to Solid methods

### LINKED ISSUE #1202: Get the control points of the Curve Loop
#### Description:

As in the title.

### PR DESCRIPTION
**Pull request by michal-pekacki**
### Issues addressed by this PR

Closes #1202 

- added `LinkedElementsInView` method to get linked elements from given view
- added `Tessalate` method to convert `CuveLoop` to list of points
- extend `Solid` method to get the `Solid` from `ViewPlan`

### Additional comments

New PR for [#1203](https://github.com/BHoM/Revit_Toolkit/pull/1203)

### CODE REVIEW

File: Revit_Core_Engine/Query/LinkedElementsInView.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Get linked elements from given view.")]
```
Comment (pawelbaran) [inline Revit_Core_Engine/Query/LinkedElementsInView.cs:37]: What does it mean? Will invisible elements from link be returned? Would be good to add as much detail to description as possible.

File: Revit_Core_Engine/Query/Tessellate.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+
+        [Description("Get control points of CurveLoop with tessellation.")]
+        [Input("curveLoop", "CurveLoop to get the points from.")]
+        [Output("curveLoopPoints", "CurveLoop points.")]
+        public static List<XYZ> Tessellate(this CurveLoop curveLoop)
+        {
+            if (curveLoop == null)
```
Comment (pawelbaran) [inline Revit_Core_Engine/Query/Tessellate.cs:42]: worth checking against zero curves in the loop, also an error would be welcome

File: Revit_Core_Engine/Create/Solid.cs [hunk 3]
```diff
@@ -94,5 +94,62 @@ public static Solid Solid(IList<CurveLoop> boundaries, double bottomElevation, d
...
+                viewBBox.Min = new XYZ(-1e6, -1e6, -1e6);
+                viewBBox.Max = new XYZ(1e6, 1e6, 1e6);
+            }
+
+            //topElevation (from the CutPlane)
+            Level topLevel = doc.GetElement(viewRange.GetLevelId(PlanViewPlane.CutPlane)) as Level;
+            double topElevation;
+            if (topLevel == null)
+            {
+                topElevation = 1e6;
```
Comment (pawelbaran) [inline Revit_Core_Engine/Create/Solid.cs:122]: Sanity check: isn't that beyond Revit's acceptable range?

**Review by pawelbaran — CHANGES_REQUESTED** [review]
Good, clean code, minor changes requested

File: Revit_Core_Engine/Create/Solid.cs (hunk 3 above)
Comment (michal-pekacki) [reply Revit_Core_Engine/Create/Solid.cs:122]: I checked that and bounding boxes is created for that range

**Pull request closed without merging**